package.json

```diff
@@ -23,7 +23,7 @@
     "prepublish": "yarn run build",
     "test": "jest",
     "flow": "flow",
-    "lint": "eslint \"?(__mocks__|src|tests)/**/*.+(js|ts)\"",
+    "lint": "eslint \"?(__mocks__|src|tests)/**/*.+(js|ts)\" --fix",
```

Comment:
let's split lint from lint-fix, if ci is going to use lint, I don't think it should be fixing anything...

Reply:
that was actually not intentionally maybe that happen on merging Master